README.md

```diff
@@ -23,6 +23,11 @@ GCP_USER: Your Google Cloud Platform user (email) associated with the project.
 GCP_KEY: The path to your GCP service account key file (JSON file).
 GCP_PROJECT_NAME: The name of your GCP project where Secrets Manager will be used.
 
+### IMPORTANT
+The GCP_KEY in the format of -----BEGIN PRIVATE KEY----- .... -----END PRIVATE KEY----- taken from the JSON
```

Comment:
Maybe worth mentioning that Cloudflare will then transform the newlines into `\n`?

Reply:
Yes. We need to re-redact the Readme in that regard.